control/veda_file_ingest.py

```diff
@@ -131,7 +131,9 @@ def _gather_metadata(self):
             .replace('\'', '\\\'') \
             .replace('[', '\[') \
             .replace(']', '\]') \
-            .replace('&', '\&')
+            .replace('&', '\&') \
+            .replace('(', '\(') \
+            .replace(')', '\)')
 
```

Comment:
@schenedx It might make sense to make a catch-all function for these that replace them with mystery-characters. 
You could decode the chars iteratively and then just replace them with `?` or something. This is fine though.

Reply:
I know what you are saying. This is a slippery slope. 
I am not prepared to sink time into these right now. But yes, this is one of the area in the code we should improve.